src/libs/PolicyUtils.js

```diff
@@ -0,0 +1,63 @@
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+
+/**
+ * Checks if we have any errors stored within the POLICY_MEMBER_LIST. Determines whether we should show a red brick road error or not.
+ * Data structure: {email: {role:'bla', errors: []}, email2: {role:'bla', errors: [{1231312313: 'Unable to do X'}]}, ...}
```

Comment:
```suggestion
 * Data structure: {email: {role:'user', errors: []}, email2: {role:'admin', errors: [{1231312313: 'Unable to do X'}]}, ...}
```

Reply:
Ha ok sounds good. This was not my comment originally, but I'll update it.

Reply:
+1